tasks/rx-install.yml

```diff
@@ -185,65 +185,65 @@
 - name: Deploy RecoverX Node Template
   template:
     src: nodes.j2
-    dest: "{{ rdio_user_home }}/nodes"
-    owner: "{{ rdio_user }}"
-    group: "{{ rdio_user }}"
+    dest: "{{ mosaic_user_home }}/nodes"
+    owner: "{{ mosaic_user }}"
+    group: "{{ mosaic_user }}"
   become: true
-  become_user: "{{ rdio_user }}"
+  become_user: "{{ mosaic_user }}"
   when: inventory_hostname == groups['rx'][0]
 
 - name: Detect and register Nodes
   slurp:
-    src: "{{ rdio_user_home }}/nodes"
+    src: "{{ mosaic_user_home }}/nodes"
   register: nodes
   when: inventory_hostname == groups['rx'][0]
 
 - name: Detect existing RecoverX installation
   stat:
-    path: "{{ rdio_installer_directory }}/datosinstall"
-  register: rdio_dir
+    path: "{{ mosaic_installer_directory }}/datosinstall"
+  register: mosaic_dir
   changed_when: False
   ignore_errors: True
 
-- name: Create/verify Installer directory {{ rdio_installer_directory }}
+- name: Create/verify Installer directory {{ mosaic_installer_directory }}
   file:
-    path: "{{ rdio_installer_directory }}"
+    path: "{{ mosaic_installer_directory }}"
     state: directory
     mode: 0700
-  become_user: "{{ rdio_user }}"
+  become_user: "{{ mosaic_user }}"
 
-- name: Copy RDIO release file to {{ groups['rx'][0] }} - {{ rdio_install_file }}
+- name: Copy Mosaic release file to {{ groups['rx'][0] }} - {{ mosaic_install_file }}
   unarchive:
-    src: "{{ rdio_install_file }}"
-    dest: "{{ rdio_installer_directory }}/"
-    owner: "{{ rdio_user }}"
-    group: "{{ rdio_user }}"
+    src: "{{ mosaic_install_file }}"
+    dest: "{{ mosaic_installer_directory }}/"
+    owner: "{{ mosaic_user }}"
+    group: "{{ mosaic_user }}"
     list_files: true
   become: true
-  become_user: "{{ rdio_user }}"
+  become_user: "{{ mosaic_user }}"
   register: deploy_files
   when: 
     - inventory_hostname == groups['rx'][0]
-    - not rdio_dir.stat.exists
+    - not mosaic_dir.stat.exists
 
-- name: Install RecoverX - {{ rdio_install_file }}
+- name: Install RecoverX - {{ mosaic_install_file }}
   shell: >
-    ./install_datos
-    -i {{ nodes['content'] | b64decode }}
-    -t {{ rdio_installer_directory }}/datosinstall
+    ./install_datos 
+    -i {{ nodes['content'] | b64decode | trim }} 
+    -t {{ mosaic_installer_directory }}/datosinstall 
     --skip-eula-check
   args:
-    chdir: "{{ rdio_installer_directory }}/{{ deploy_files.files[0] }}"
+    chdir: "{{ mosaic_installer_directory }}/{{ deploy_files.files[0] }}"
   become: true
-  become_user: "{{ rdio_user }}"
-  register: rdio_install
+  become_user: "{{ mosaic_user }}"
+  register: mosaic_install
   when:
-    - not rdio_dir.stat.exists
+    - not mosaic_dir.stat.exists
     - inventory_hostname == groups['rx'][0]
 
 - name: Deploy RecoverX init script
```

Comment:
Should this say Mosaic?

Reply:
Says mosaic in my copy. The next push should fix this.